web/client/epics/layers.js

```diff
@@ -148,8 +149,25 @@ const updateSettingsParamsEpic = (action$, store) =>
             );
         });
 
+/**
+ * Triggers updates of the layer property of widgets on layerFilter change
+ * @memberof epics.layers
+ * @param {external:Observable} action$ manages `CHANGE_LAYER_PROPERTIES`
+ * @return {external:Observable}
+ */
+const changeLayerPropertiesEpic = (action$, store) =>
```

Comment:
move this epic in the widgets file, because is an effect of that concerns widgets, not layers